dashboard/app/helpers/levels_helper.rb

```diff
@@ -126,6 +125,12 @@ def app_options
       callouts: callouts
     )
 
+    # External project levels are any levels of type 'external' which use
+    # the projects code to save and load the user's progress on that level.
+    view_options(is_external_project_level: true) if @level.pixelation?
```

Comment:
Now that we have `is_channel_backed` below do we still need `is_external_project_level`?